core/trace/src/main/scala/org/typelevel/otel4s/trace/Tracer.scala

```diff
@@ -137,6 +142,7 @@ object Tracer {
       private val builder = SpanBuilder.noop(noopBackend)
       private val resourceUnit = Resource.unit[F]
       val meta: Meta[F] = Meta.disabled
+      val currentSpan: F[Span[F]] = builder.startUnmanaged
```

Comment:
As an alternative, we can use `Applicative[F].pure(Span.fromBackend(noopBackend))`